src/pages/Home/index.tsx

```diff
@@ -1,21 +1,37 @@
 
-import { useState } from "react";
-import { Button, Input, Title } from "_/components";
-import { useAuth } from "_/contexts";
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Button, Title } from "_/components";
+import { Calendar } from "_/components/Calendar";
+import { ROUTE_PATHS } from "_/constants";
+import { useAuth, useBooking, useCheckin } from "_/contexts";
+import { parseEvents } from "_/helpers";
+import { Event } from "_/models";
 
 export function Home(){
-    const [value, setValue] = useState('')
+    const [events, setEvents] = useState([] as Array<Event>)
```

Comment:
const [events, setEvents] = useState<Array<Event>>([]), you could write it like this too :)

Reply:
done ✅